### PR TITLE
307 implement withdraw instruction

### PR DESCRIPTION
**Pull request by ivandzen**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ARG SOLANA_REVISION=v1.8.12-testnet
-ARG EVM_LOADER_REVISION=latest
+ARG EVM_LOADER_REVISION=ci-308-account_v2
```
Comment (gigimon) [inline Dockerfile]: change this before merge

File: proxy/testing/test_neon_token.py [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+
+    function withdraw(bytes32 spender) public payable {
+        (bool success, bytes memory returnData) = NeonPrecompiled.delegatecall(abi.encodeWithSignature("withdraw(bytes32)", spender));
+        require(success);
+    }
+}
+'''
+
+
+proxy_url = os.environ.get('PROXY_URL', 'http://127.0.0.1:9090/solana')
```
Comment (gigimon) [inline proxy/testing/test_neon_token.py]: this's constants must be in uppercase

File: proxy/testing/test_neon_token.py [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+        print(f'withdraw_tx_hash: {withdraw_tx_hash.hex()}')
+        withdraw_tx_receipt = proxy.eth.wait_for_transaction_receipt(withdraw_tx_hash)
+        print(f'withdraw_tx_receipt: {withdraw_tx_receipt}')
+        print(f'deploy status: {withdraw_tx_receipt.status}')
+    
+    def test_success_withdraw_to_non_existing_account(self):
+        """
+        Should succesfully withdraw NEON tokens to previously non-existing Associated Token Account
+        """
+        dest_acc = SolanaAccount()
```
Comment (gigimon) [inline proxy/testing/test_neon_token.py]: We can move 
```python
dest_acc = SolanaAccount()
solana.request_airdrop(dest_acc.public_key(), 1000_000_000_000)
```
to setup method
https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUp

**Review by gigimon — CHANGES_REQUESTED** [review]
We need a cases like:
1. Withdrawal more than exist in account
2. withdraw all balance (if we can't pay for gas)

File: Dockerfile (hunk 1 above)
Comment (ivandzen) [reply Dockerfile]: Fix

File: proxy/testing/test_neon_token.py (hunk 2 above)
Comment (ivandzen) [reply proxy/testing/test_neon_token.py]: Fixed

File: proxy/testing/test_neon_token.py (hunk 3 above)
Comment (ivandzen) [reply proxy/testing/test_neon_token.py]: I've decided to create new accounts for every test

**Review by gigimon — APPROVED** [review]

**Pull request merged**